src/index.html

```diff
@@ -7,11 +7,184 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Personal information</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
+
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="labelWithMargin">
+          Surname:
+          <input
+            type="text"
+            name="usersurname"
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label for="name" class="labelWithMargin">
+          Name:
+          <input
+            type="text"
+            name="username"
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label for="old" class="labelWithMargin">
+           How old are You?
+          <input
+            type="number"
+            name="userage"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label for="old" class="labelWithMargin">
```

Comment:
![image](https://user-images.githubusercontent.com/54936044/227183641-3435efbc-6a94-4687-9c9a-4436b92e6b4f.png)
Please check for all cases!